src/webgpu/api/validation/copy_between_linear_data_and_texture/README.txt

```diff
@@ -24,4 +24,5 @@ Test coverage:
 	- texel_block_alignment_on_size: for all formats and coordinates.
 	- texture_range_conditons: for all coordinate and various combinations of origin, copyExtent, textureSize and mipLevel.
 
-TODO: more test coverage for 1D and 3D textures.
\ No newline at end of file
+TODO: more test coverage for 1D and 3D textures.
+TODO: add tests for buffer usage and bufferCopyView
```

Comment:
Can you detail what needs testing in buffercopyview?

Reply:
Done